github_pr.py

```diff
@@ -237,8 +255,10 @@ def main():
         github_update_pr(**args)
 
     gh = Github(args['token'])
-    if not args['numberonly'] and not args['noratelimit']:
-        print "Github Rate Limiting: %d remaining of max %d" % (gh.rate_limiting[0], gh.rate_limiting[1])
+
+    if (('numberonly' in args) and not args['numberonly']):
```

Comment:
@cchristous removed the else, we still need the double negative, it doesnt work by changing to "store_false"